KurakuAPI/Services/AccomodationService.cs

```diff
@@ -0,0 +1,23 @@
+﻿using DataAccess;
+using KurakuAPI.Models;
+using KurakuAPI.Services.Interfaces;
+using Microsoft.EntityFrameworkCore;
+
+namespace KurakuAPI.Services;
+
+public class AccomodationService(ApplicationDbContext dbContext) : IAccomodationService
+{
+    public IQueryable<AccomodationModel> GetAccomodations()
+    {
+        return dbContext.Accomodations
+            .Include(a => a.Itinerary)
```

Comment:
I am curios as to why we are including `Itinerary` on this projection.

Reply:
{
  "id": 0,
  "airLine": "string",
  "flightNumber": "string",
  "departureTime": "2024-11-19T07:43:46.270Z",
  "arrivalTime": "2024-11-19T07:43:46.270Z",
  "fromLocation": "string",
  "toLocation": "string",
  "itineraryId": 0,
  "itinerary": {
    "id": 0,
    "destination": "string",
    "userId": "string",
    "user": {
      "id": "string",
      "userName": "string",
      "normalizedUserName": "string",
      "email": "string",
      "normalizedEmail": "string",
      "emailConfirmed": true,
      "passwordHash": "string",
      "securityStamp": "string",
      "concurrencyStamp": "string",
      "phoneNumber": "string",
      "phoneNumberConfirmed": true,
      "twoFactorEnabled": true,
      "lockoutEnd": "2024-11-19T07:43:46.270Z",
      "lockoutEnabled": true,
      "accessFailedCount": 0
    },
    "startDate": "2024-11-19T07:43:46.270Z",
    "endDate": "2024-11-19T07:43:46.270Z"
  }
}

I want my flight output to be exactly like that, that's why I included an Itinerary entity for my flight get request.

Reply:
If it's not necessary then I'll just not include it.